creator-node/src/apiHelpers.js

```diff
@@ -56,8 +56,11 @@ module.exports.handleResponseWithHeartbeat = (func) => {
       sendResponseWithHeartbeatTerminator(req, res, resp)
       next()
     } catch (error) {
-      console.error('HandleResponse', error)
-      next(error)
+      sendResponseWithHeartbeatTerminator(
+        req,
+        res,
+        errorResponse(500, error)
+      )
```

Comment:
whats reason for change?

Reply:
there's no next(error) left in the handlers, so this error isn't gracefully returned to client. The response here should also be terminated with the heartbeat end quote.

as it stands, the client gets back
```
{
  _h: "11111 status: 200, data: { ... }
}
```

rather than
```
{
  _h: "11111",
  status: 200,
  data: { ... }
}
```

Reply:
oh interesting, sg